### DISCUSSION TITLE
My application is is not getting approved.

### DISCUSSION THREAD

**Discussion by sdimi98** [community/community#113532] [GitHub Education]
### Hi everyone,

Hello, I keep trying to apply for the educational pack, but it keeps getting rejected. I have included all the details in my application. Maybe the photo quality is not good enough? I am using an iPhone and the website is only detecting my selfie camera (which  I tried using). Any help would be greatly appreciated.

**Comment by samus-aran** [comment]
Hi @sdimi98  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by sdimi98** [reply]
![image](https://github.com/community/community/assets/147751237/4646dcc4-5b62-4b9d-93e2-cfe81ce30af7)

This is basically the error I am getting and I assume it is automated. The weird part is that I have classmates that managed to get it done and we are all following the same instructions to get it. I tried using a different browser and also my phone to verify.

**Reply by samus-aran** [reply]
Thanks for this!

The first rejection reason you need to address this as a priority.

- Please ensure that your browser settings allow location verification. You may alternatively use your phone if your computer settings do not allow this.
- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out  [this discussion](https://github.com/orgs/community/discussions/42246) on assessing your hardware or operating system's settings. 
- To troubleshoot the location check further, you may wish to try disabling your add-ons and/or clearing your cookies and cache, or try another browser. 💡 

The second rejection reason is that the application can't detect your school name. Is it translated? Is it a Logo? Ensure when you take an image of your evidence that it's legible.

Hope this helps 👍

**Reply by sdimi98** [reply]
![image](https://github.com/community/community/assets/147751237/5490441b-6c49-4c5a-bbb9-a6c6e8efdefd)
The location settings are enabled both on my system and on my browser, but I get that error regardless. I am using Windows 11.  I have already tried a different browser.
